src/components/TextEditor/TextEditor.tsx

```diff
@@ -7,58 +7,47 @@ import AddFile from '../../assets/addfile.png';
 import AddImg from '../../assets/img.png';
 import AddList from '../../assets/list.png';
 
-interface Props {
-    getObjectNote: (obj: Note) => void;
-}
-interface Note {
-    title: string;
-    description: string;
-}
-
-const TextEditor = ({ getObjectNote }: Props): React.ReactElement => {
+const TextEditor = (): React.ReactElement => {
     const [descriptionValue, setDescriptionValue] = useState('');
     const [titleValue, setTitleValue] = useState('');
-    const [selectedValue, setSelectedValue] = useState('');
+    const [fontSize, setFontSize] = useState('');
     const [onClickIcon, setOnClickIcon] = useState(false);
     const [uploadedImg, setUploadedImg] = useState('');
-    const [styles, setStyles] = useState<CSSProperties>({});
-    const [, setObjNote] = useState({});
+    const [styles, setStyles] = useState<CSSProperties>({
+        fontSize: '20px',
+        textAlign: 'left',
+    });
+    const [objNote, setObjNote] = useState({});
 
     const uploadedImage = (event: React.ChangeEvent<HTMLInputElement>): void => {
         const files: FileList | null = event.currentTarget.files;
         if (files) Array.from(files).map((file: { name: string }) => setUploadedImg(URL.createObjectURL(file)));
     };
 
-    const getSelectedValue = (event: React.ChangeEvent<HTMLSelectElement>): void => {
-        setSelectedValue(event.target.value);
+    const getFontSizeValue = (event: React.ChangeEvent<HTMLSelectElement>): void => {
+        setFontSize(event.target.value);
         setStyles({ ...styles, fontSize: `${event.target.value}px` });
     };
 
-    const handleAlignTextLeft = (): void => {
-        setStyles({ ...styles, textAlign: 'left' });
-    };
-
-    const handleAlignTextCenter = (): void => {
-        setStyles({ ...styles, textAlign: 'center' });
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Do not use any

Reply:
Can you help me with the type of that function, pls